docs/getting-started/smr-disk.md

```diff
@@ -115,24 +115,24 @@ sd 5:0:0:0: [sdb] Attached SCSI disk
 ...
 ```
 
-Among other information, the zone model of the disk is confirmed as
-host managed. The total number of zones of the disk is also displayed. In this
-example, the disk capacity is 15 TB and has 55880 zones.
+The zone model of the disk is confirmed to be "host managed". The total number
+of zones on the disk is also displayed. In this example, the disk capacity is
+15 TB and the disk has 55880 zones.
 
-The zone size of the disk can also be inspected through sysfs, with the
-attribute *chunk_sectors*.
+The zone size of the disk can be inspected by using sysfs to examine the
+attribute *chunk_sectors*:
```

Comment:
This is very convoluted... Maybe something simpler like this ?

The zone size of the disk can be discovered using the *chunk_sectors* sysfs attribute file:

Reply:
I see your point.

I've tried to make it yet less convoluted.

Let me know what you think.